### PR TITLE
update projects to new csproj format (vs 2017)

### PR DESCRIPTION
**Pull request by Styxxy**
fixes issue #188

### CODE REVIEW

**Comment by Styxxy** [comment]
@kevingy Do you have time to look at this?

File: SimpleBrowser/SimpleBrowser.csproj [hunk 1]
```diff
@@ -1,139 +1,49 @@
...
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <Description>SimpleBrowser is a lightweight, yet highly capable browser automation engine designed for automation and testing scenarios.</Description>
+    <Copyright></Copyright>
+    <AssemblyTitle>SimpleBrowser</AssemblyTitle>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <VersionPrefix>0.5.79</VersionPrefix>
+    <Authors>Nathan Ridley, Teun Duynstee, Kevin Yochum, Joe Feser</Authors>
+    <TargetFrameworks>net40</TargetFrameworks>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (kevingy) [inline SimpleBrowser/SimpleBrowser.csproj]: It looks like the new project file format may have changed the way TreatWarningsAsErrors works. When I opened the solution in VS2017, the "Treat Warnings As Errors" setting for the SimpleBrowser project was set to "Specific Warnings" and set to ignore "NU1605". I changed this setting to "All". When I did that, the following was added to the project file:
```xml
<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net40|AnyCPU'">
    <WarningsAsErrors />
</PropertyGroup>
```
We'll need this setting in the new project file - treating all warnings as errors.

Comment (kevingy) [reply SimpleBrowser/SimpleBrowser.csproj]: This modification would be needed in the unit test and sample application project files as well.

**Comment by kevingy** [comment]
I may have also introduced a merge conflict in this pull request. Can you resolve that, please?

File: SimpleBrowser/SimpleBrowser.csproj (hunk 1 above)
Comment (Styxxy) [reply SimpleBrowser/SimpleBrowser.csproj]: Added the item to all csprojs, without condition of configuration (release/debug).

**Comment by Styxxy** [comment]
I will add the netstandard later, in another PR. When specifying multiple target frameworks, it will always build all the specified targets. However in VS, the code view will be targetted to the first specified framework.

**Comment by Styxxy** [comment]
I fixed 2 unit tests for NUnit 3, they changed ttheir "run directory" so relative paths in unit tests don't work anymore.

There's still 1 unit test that fails (RefererHeader.When_Testing_Referer_Unsafe_Url_Secure_Transition) however it doesn't seem to be caused by my changes.

**Comment by kevingy** [comment]
That unit test is definitely not you. We don't have our own web servers that we can configure and test against. As a result, we're forced to test against public web sites. This is a case where the public web site changed. As a result, our test broke.

I foresee that changing sometime this year. I'll be releasing a product I've been working on. When that happens, I'll volunteer some resources on the server for SimpleBrowser testing.

As far as this pull request goes, I should be able to finish testing and merge the request later today.

**Pull request merged**